eos/constraints/B-to-P-form-factors.yaml

```diff
@@ -736,6 +736,31 @@ B->K::FormFactors[f_+,f_T]@KR:2017A:
         -   [0.459517, 1.0     , 0.0     , 0.0     ]
         -   [0.0     , 0.0     , 1.0     , 0.058965]
         -   [0.0     , 0.0     , 0.058965, 1.0     ]
+B->K::FormFactors[parametric,BFW]@GRvDV:2023A:
+    type: MultivariateGaussian(Covariance)
+    observables:
+      [ 'B->K::a^f+_0@BFW2010', 'B->K::a^f+_1@BFW2010', 'B->K::a^f+_2@BFW2010',
+                                'B->K::a^f0_1@BFW2010', 'B->K::a^f0_2@BFW2010',
+        'B->K::a^fT_0@BFW2010', 'B->K::a^fT_1@BFW2010', 'B->K::a^fT_2@BFW2010' ]
+    kinematics:
+      [ {}, {}, {}, {}, {}, {}, {}, {} ]
+    options:
+      [ {'B->K::tp@BFW2010': 30.261001}, {'B->K::tp@BFW2010': 30.261001}, {'B->K::tp@BFW2010': 30.261001},
```

Comment:
I think this is correct. However, the Github syntax highlighting is playing a prank on me. @mreboud can you confirm that the syntax is correct here? See the following:
![image](https://user-images.githubusercontent.com/42914/237051323-0ea7df08-7fee-4be9-819d-8d7e236208ba.png)

Reply:
This is correct, gh is lost

Reply:
this is due to the `:`

Reply:
Very good. Will merge!